### PR TITLE
feat: Node File System Persister

### PR DESCRIPTION
**Pull request by offirgolan**
- File System Persister
- Node tests

screen shot 2018-07-10 at 6 38 56 pm

### CODE REVIEW

File: docs/persisters/fs.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+## Options
+
+### recordingsDir
+
+_Type_: `String`
+_Default_: `'recordings'`
+
+The root directory to store all recordings.
```
Comment (jasonmit) [inline docs/persisters/fs.md]: Are absolute paths supported as well?  Might want to mention

File: packages/@pollyjs/core/tests/unit/polly-test.js [hunk 2]
```diff
@@ -132,12 +131,7 @@ describe('Unit | Polly', function() {
     setupPolly({ adapters: [] });
 
     it('should not be configurable once requests are handled', async function() {
-      const { server } = this.polly;
-
-      this.polly.configure({ adapters: ['fetch'] });
-      server.get('/ping').intercept((req, res) => res.status(200));
-
-      expect((await fetch('/ping')).status).to.equal(200);
+      this.polly._requests.push({});
```
Comment (jasonmit) [inline packages/@pollyjs/core/tests/unit/polly-test.js]: Why this change vs it handling a request?

File: packages/@pollyjs/node-server/src/api.js [hunk 3]
```diff
@@ -2,8 +2,8 @@ import fs from 'fs-extra';
 import path from 'path';
 
 export default class API {
-  constructor(recordingsDir) {
-    this.recordingsDir = recordingsDir;
+  constructor(options = {}) {
+    this.recordingsDir = options.recordingsDir;
```
Comment (jasonmit) [inline packages/@pollyjs/node-server/src/api.js]: Should we throw if no recordingDir?

File: packages/@pollyjs/persister-fs/src/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  findRecording(recordingId) {
+    return this.api.getRecording(recordingId).body || null;
+  }
+
+  saveRecording(recordingId, data) {
+    /*
+      Pass the data through the base persister's stringify method so
+      the output will be consistent with the rest of the persisters.
+    */
+    this.api.saveRecording(recordingId, parse(this.stringify(data)));
```
Comment (jasonmit) [inline packages/@pollyjs/persister-fs/src/index.js]: Any reason for no `return` since saveRecording does have a return value

File: packages/@pollyjs/persister-fs/src/index.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  deleteRecording(recordingId) {
+    this.api.deleteRecording(recordingId);
```
Comment (jasonmit) [inline packages/@pollyjs/persister-fs/src/index.js]: Same as above

File: packages/@pollyjs/persister-fs/src/index.js (hunk 4 above)
Comment (offirgolan) [reply packages/@pollyjs/persister-fs/src/index.js]: The other persisters don't return anything from `saveRecording`.

File: packages/@pollyjs/persister-fs/src/index.js (hunk 5 above)
Comment (offirgolan) [reply packages/@pollyjs/persister-fs/src/index.js]: The other persisters don't return anything from `deleteRecording `.
